lib/model/migrations/20230324-01-edit-dataset-verbs.js

```diff
@@ -0,0 +1,33 @@
+// Copyright 2023 ODK Central Developers
+// See the NOTICE file at the top-level directory of this distribution and at
+// https://github.com/getodk/central-backend/blob/master/NOTICE.
+// This file is part of ODK Central. It is subject to the license terms in
+// the LICENSE file found in the top-level directory of this distribution and at
+// https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+// including this file, may be copied, modified, propagated, or distributed
+// except according to the terms contained in the LICENSE file.
+
+const { without } = require('ramda');
+
+/* eslint-disable no-await-in-loop */
+
+const up = async (db) => {
+  // grant rights.
+  for (const system of [ 'admin', 'manager', 'viewer' ]) {
+    const [{ verbs }] = await db.select('verbs').from('roles').where({ system });
+    const newVerbs = verbs.concat([ 'dataset.read', 'entity.read' ]);
```

Comment:
Do you want to throw in `entity.create` and `entity.update` while you're here? Maybe we'd need one other one for updating the properties of an entity, though maybe not. Looking at the existing submission verbs, I only see list/read/create/update.

Reply:
I can! Just need to figure out what roles does `entity.create`/`entity.update` apply to? Probably not `viewer`. Does that just leave `admin` + `manager`?

Reply:
I agree, seems like it would just be `admin` + `manager` for now. I guess that would maybe add complications since the migration also deals with viewers. Those verbs could also come in a separate PR, whatever's more convenient!

Reply:
Doing it here so they can be used soon, even though there's nothing that really tests these new verbs yet!